package.json

```diff
@@ -52,9 +53,10 @@
     "postcss": "^8.4.34",
     "prettier": "^3.2.5",
     "prettier-plugin-tailwindcss": "^0.5.11",
-    "prisma": "^5.10.2",
+    "prisma": "^4.4.0",
     "tailwindcss": "^3.4.1",
-    "typescript": "^5.3.3"
+    "typescript": "^5.3.3",
+    "node": "^18.20.0"
```

Comment:
I think this doesn't belong here. Node is typically a globally installed package, not project level.

Reply:
Removed this